homeassistant/components/group/entity.py

```diff
@@ -153,7 +154,7 @@ def __init__(
         self._attr_name = name
         self._state: str | None = None
         self._attr_icon = icon
-        self._set_tracked(entity_ids)
+        self._entity_ids = entity_ids
```

Comment:
Instead of processing the entity_ids at init we now process them when the entity is to be added to hass. This way we have an `entity_id` we can use.